src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -1,15 +1,39 @@
 import React from "react";
 import PropTypes from "prop-types";
 
+import styledSystemPropTypes from "@styled-system/prop-types";
 import tagComponent from "../../../utils/helpers/tags";
 import Fieldset from "../../../__internal__/fieldset";
 import RadioButtonGroupStyle from "./radio-button-group.style";
 import RadioButtonMapper from "./radio-button-mapper.component";
 import Logger from "../../../utils/logger/logger";
 import useIsAboveBreakpoint from "../../../hooks/__internal__/useIsAboveBreakpoint";
+import filterObjectProperties from "../../../utils/helpers/filter-object-properties";
 
 let deprecatedWarnTriggered = false;
 
+const marginProperties = [
+  "margin",
+  "m",
+  "marginLeft",
+  "ml",
+  "marginRight",
+  "mr",
+  "marginTop",
+  "mt",
+  "marginBottom",
+  "mb",
+  "marginX",
+  "mx",
+  "marginY",
+  "my",
+];
+
+const marginPropTypes = filterObjectProperties(
```

Comment:
Can we move this and `marginProperties` into a reusable util somewhere? I think we'll need it in other components as well

Reply:
this is already a reusable util, you'll just have to specify what you want to be filtered out. I don't think that we need an separate util for margin and padding as it could be easily done by just specifying a second argument.

Reply:
I disagree because we're going to use this `marginPropTypes` in lots of components! and this is just going to be duplicated like 50+ times. 

At the very least I think 
```
const marginProperties = [
  "margin",
  "m",
  "marginLeft",
  "ml",
  "marginRight",
  "mr",
  "marginTop",
  "mt",
  "marginBottom",
  "mb",
  "marginX",
  "mx",
  "marginY",
  "my",
];
```
should be moved into a constants file

Reply:
ok, you've pointed the wrong line, I thought you're talking about `marginPropTypes`

Reply:
I have moved `marginProperties` to a separate file

Reply:
I was talking about both 😉